<launch>
    <node name="move_grab" type="move_grab_v3.py" pkg="cobot_arm" output="screen">
        <rosparam command="load" file="$(find cobot_nav)/config/record_pose.yaml"/>
    </node>
</launch>